source/blog/2017-03-30-minishift-intro.html.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Introducing Minishift - Run OpenShift locally
+date: 2017-03-30 05:45 UTC
+author: budhram
+published: true
+comments: true
+tags: minishift, atomic-developer-bundle, openshift-local
+---
+
+It has been quite a few months since our [last blog post](http://www.projectatomic.io/blog/2016/08/vagrant-service-manager-1-3-0-release/) on [vagrant-service-manager](https://github.com/projectatomic/vagrant-service-manager) as we have been looking around for alternative solutions to replace the [Atomic Developer Bundle (ADB)](https://github.com/projectatomic/adb-atomic-developer-bundle). A separate blog post will address the reasons for this.
```

Comment:
I do not think we need to mention about our last blog post on VSM. We should communicate that we have came  up with a better user experience in Minishift as compared to ADB. Also the sentence about "have been looking around for alternative solutions to replace the ADB" is wrong. We wanted to improve the user experience and  also wanted to address the issues users were facing with ADB.